NGitLab/Impl/IssueClient.cs

```diff
@@ -192,6 +194,16 @@ public Task<Issue> CloneAsync(int projectId, int issueIid, IssueClone issueClone
         return _api.Post().With(issueClone).ToAsync<Issue>(string.Format(CultureInfo.InvariantCulture, CloneIssueUrl, projectId, issueIid), cancellationToken);
     }
 
+    public IEnumerable<Participant> GetParticipants(int projectId, int issueIid)
```

Comment:
Could you add a test to validate the retrieval of participant and the un-subscription on an issue?

Reply:
Lot of stuff is missing to test correctly the subscription. 
I add a test to retrieves participant